build/Dockerfile

```diff
@@ -9,15 +9,11 @@ ENV OPERATOR=/usr/local/bin/keycloak-operator \
     USER_UID=1001 \
     USER_NAME=keycloak-operator
 
-ENV SHA1_FILE=/BUILD_SHA1.txt
```

Comment:
To be honest, I liked the approach with those 2 files. It's very convenient, you can `ssh` (or `rsh`) to a running container and find out what version your on in just a second.